### DISCUSSION TITLE
Issues in feature extractions in OpenVSLAM

### DISCUSSION THREAD

**Discussion by surfii3z** [stella-cv/stella_vslam#61] [Show and tell]
Hi all

I have been working on Visual SLAM for quite a while and I have the idea to use Neural Network, [packnet-sfm](https://github.com/surfii3z/packnet_sfm_ros) in particular, to infer the depth and use the SLAM as RGB-D system instead of just monocular system. 

By doing so, I hope to solve a few challenges of the monocular SLAM system, namely scale ambiguity, pure-rotation and map initialization.

I pose this [Youtube link](https://youtu.be/qNbfIrcvDEM?t=63) to show some result of my experiment on KITTI odometry sequence 07. I found that OpenVSLAM has some issues.

In the video, OpenVSLAM has a lot of features flickering on the road, in contrast to ORB_SLAM2. After that, it also failed to track the features after the car was turning, whereas ORB_SLAM2 works just fine.

I wonder if we cloud identify/ solve these problems of OpenVSLAM.

Best

Surf

**Comment by ymd-stella** [comment]
It's related to https://github.com/OpenVSLAM-Community/openvslam/discussions/73.
